featuretools/io/sql.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+import sqlalchemy as sa
+
+import featuretools as ft
+
+
+def from_sql(id, connection=None, connection_string=None, tables=None, index={},
```

Comment:
need to add a docstring for this method. Look at any some of our public methods for the formatting (`encode_features` is a good one to look at). Would be nice to include an example section based on one of the test cases, as well. 

Also, add this method to `api_reference.rst` in a new section title "Input/Output". By including it the api reference, a page will be automatically for it when build the docs, so you can see how it looks. Can you also somehow indicate that this method is "Beta" when you add?